### ISSUE TITLE
Support for macros dependencies

### ISSUE THREAD

**Issue by l4l** [CoatiSoftware/CoatiBugTracker#88]
It's a nice feature for Coati to support macros dependency. E.g. from systemd:

![screenshot- - -10-16 48 44-msk-2016](https://cloud.githubusercontent.com/assets/5658339/15966324/b89e3e48-2f33-11e6-98be-0ba79b2ff03a.png)

Is it possible to define another macro USEC_PER_SEC as dependency to the JOBS_IN_PROGRESS_PERIOD_USEC?

**Comment by egraether** [comment]
Yes, I agree that would be a nice feature. Macro dependency works only on file level so far.
